avocado/core/job.py

```diff
@@ -483,7 +483,7 @@ def _run(self):
 
         if 'INTERRUPTED' in summary:
             self.exitcode |= exit_codes.AVOCADO_JOB_INTERRUPTED
-        if 'FAIL' in summary:
+        if 'FAIL' in summary and not getattr(self.args, 'relax_exit_code', False):
```

Comment:
As expressed before, I think, as much as possible, we should try to keep Avocado reporting errors in the exit status... Ignoring something should, AFAICT, something the caller should do.

But since you guys have much more experience with say, running Avocado under Jenkins, there may be justifications for adding something like that to Avocado itself.  I'll ask you to remove this for now, and open a proper discussion on the maling list about it.  A well discussed and informed discussion seems to be the best way out of this.